src/docs/asciidoc/nav.adoc

```diff
@@ -0,0 +1,9 @@
+* xref:dedication.adoc[]
+* xref:acknowledgements.adoc[]
```

Comment:
I recently learned this is spelled "acknowledgments". I can rename the file after this is merged.